resources/scss/modules/_card.scss

```diff
@@ -26,10 +27,26 @@
     @include card-outline;
   }
 
+  &.card-sm {
```

Comment:
We're going for [rscss](https://rscss.io/) for the css code style and structure. While it's a progressive refactor, could you name it `.-small`?